src/components/about/Button.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Button = () => {
+  return (
+    <div className="flex flex-col justify-center items-center bg-hsp-yellow text-white text-2xl font-bold font-montserrat rounded-[500rem] w-56 h-10 my-4 hover:scale-110">
```

Comment:
don't do rounded-500 rem. Change all these values to be tailwind. Not hard coded.